src/test/java/com/wikia/webdriver/elements/fandom/components/HeroBlock.java

```diff
@@ -1,5 +1,6 @@
 package com.wikia.webdriver.elements.fandom.components;
 
+import com.wikia.webdriver.common.logging.PageObjectLogging;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.wikia.webdriver.common.logging.PageObjectLogging'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)